presto-docs/src/main/sphinx/admin/function-namespace-managers.rst

```diff
@@ -0,0 +1,76 @@
+===========================
+Function Namespace Managers
+===========================
+
+.. warning::
+
+    This is an experimental feature being actively developed. The way
+    Function Namespace Managers are configured might be changed.
+
+Function namespace managers support storing and retrieving user-defined
+functions with a persistent storage, allowing users to create, alter,
+and delete functions, as well as to reference them within queries.
+
+Function Namespace
+------------------
+
+A function namespace is a catalog-schema name (i.e. ``example.test``)
+in which each function can be uniquely identified by its signature,
```

Comment:
signature includes return type and other information as well. So just say "can be uniquely identified by its name and parameter type list".